README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Hey! You need a better MD file. This is the first page people see, and so it shouldn't be like this. Please use the standard CoderSchool format, which goes something like this:

# App Title

Created by: Name

screenshot 

## Description

Description

## Features